### ISSUE TITLE
Can some one recurrence the effect on affectnet

### ISSUE THREAD

**Issue by hyfen2009** [Talented-Q/POSTER_V2#11]
I can not recurrence the experiment on affectnet7 or affectnet8, are there some details which not explain?

**Comment by shenyujie1125** [comment]
I tested it with the pre-trained model he provided, and the results were less than 67%, only 65%

**Comment by Talented-Q** [comment]
![DJQIUDQ3KV DALI5SR8D$I1](https://github.com/Talented-Q/POSTER_V2/assets/79841528/aaf72d89-f73d-4548-b133-79bc60aae085)
![OW 2X9}8%2Q)5LQCIL~H29P](https://github.com/Talented-Q/POSTER_V2/assets/79841528/0ecf2bb6-f409-4025-af40-963b4160dde1)
Hello, are there any settings that you haven't set properly? We tested according to readme and there were no issues.

**Comment by Talented-Q** [comment]
We will soon provide a link to the relevant dataset.

**Comment by balabala-h** [comment]
> ![DJQIUDQ3KV DALI5SR8D$I1](https://user-images.githubusercontent.com/79841528/256736769-aaf72d89-f73d-4548-b133-79bc60aae085.png) ![OW 2X9}8%2Q)5LQCIL~H29P](https://user-images.githubusercontent.com/79841528/256736958-0ecf2bb6-f409-4025-af40-963b4160dde1.jpg) Hello, are there any settings that you haven't set properly? We tested according to readme and there were no issues.

can you provide a train accuracy curve of affectnet . I just got a bad result.
![image](https://github.com/Talented-Q/POSTER_V2/assets/73676575/fedb1070-f906-45de-b373-53e1a13135de)

**Comment by zerooooone** [comment]
> can you provide a train accuracy curve of affectnet . I just got a bad result. ![image](https://user-images.githubusercontent.com/73676575/263883213-fedb1070-f906-45de-b373-53e1a13135de.png)

I got the same problem, too. The train accuracy was too bad. Did you solve it now?

**Comment by balabala-h** [comment]
> I got the same problem, too. The train accuracy was too bad. Did you solve it now?



> 

no

**Comment by Talented-Q** [comment]
![J04VDBJ8R3%WJG10HYNN9}V](https://github.com/Talented-Q/POSTER_V2/assets/79841528/6f954fad-6a03-42ac-9e8f-6d6cefb0c48d)

We apologize that our mistake caused POSTER++ to train poorly in AffectNet. sampler=ImbalancedDatasetSampler(train_dataset)。
The Affectnet dataset should be oversampled during training. sampler=ImbalancedDatasetSampler(train_dataset) should be added to train_loader. We apologize that we did not point out that we had given you the wrong impression. We will make changes to the above issues in the coming work.
